ddtrace/internal/packages.py

```diff
@@ -135,6 +135,14 @@ def _package_file_mapping():
         return None
 
 
+@callonce
+def _third_party_packages() -> set:
+    from gzip import decompress
+    from importlib.resources import read_binary
+
+    return set(decompress(read_binary("ddtrace.internal", "third-party.tar.gz")).decode("utf-8").splitlines())
```

Comment:
I think it's important to document how this file was created or where it was grabbed from so that we'll be able to update it periodically.

Reply:
👍  We have an internal RFC with all the details about this file